From 67ba6859810f6699481f9a70434a437fc0a112be Mon Sep 17 00:00:00 2001
From: Tian Yang <tiany@codeaurora.org>
Date: Wed, 9 Sep 2020 15:11:58 -0700
Subject: [PATCH 225/500] net_bridge: Export br_fdb_find_rcu symbol

Export br_fdb_find_rcu symbol to be used for other module
to find fdb entry.

Change-Id: I3df88b35ddfad2d295cdd45303002c5db015ff8c
Signed-off-by: Tian Yang <tiany@codeaurora.org>
---
 net/bridge/br_fdb.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/bridge/br_fdb.c b/net/bridge/br_fdb.c
index f07705fa3da1..019456870a55 100644
--- a/net/bridge/br_fdb.c
+++ b/net/bridge/br_fdb.c
@@ -305,6 +305,7 @@ struct net_bridge_fdb_entry *br_fdb_find_rcu(struct net_bridge *br,
 {
 	return fdb_find_rcu(&br->fdb_hash_tbl, addr, vid);
 }
+EXPORT_SYMBOL_GPL(br_fdb_find_rcu);
 
 /* When a static FDB entry is added, the mac address from the entry is
  * added to the bridge private HW address list and all required ports
-- 
2.34.1

